File: portfolio/challenges/sololearn_challenges/general_challenges/popsicles/Popsicles.cs
/* The program calculates whether the number of popsicles is evenly distributed among the siblings and outputs accordingly

Finished 8/18/21 */

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace portfolio{
    class popsicles{
        static void Main(string[] args){
            int siblings, popsicles;
            siblings = Console.ToInt32(Console.ReadLine());
            popsicles = Console.ToInt32(Console.ReadLine());
            Console.write(popsicles % siblings == 0 ? "give away" : "eath them yourself");
        }
    }
} 

File: portfolio/challenges/sololearn_challenges/general_challenges/popsicles/Popsicles.java
/*
    The program calculates whether the number of popsicles is evenly distributed among the siblings and outputs accordingly

    Finished 8/18/21
*/

package challenges.sololearn_challenges.general_challenges.popsicles;

import java.util.Scanner;

public class Popsicles{

    public static void main(String[] args){

        Scanner sc = new Scanner(System.in);
        int siblings = sc.nextInt();
        int popsicles = sc.nextInt();
        
        String result = (siblings % popsicles) == 0 ? "give away" : "eat them yourself";
        System.out.println(result);
        sc.close();
    }
}